keyboards/comet46/keymaps/default/keymap.c

```diff
@@ -0,0 +1,159 @@
+// this is the style you want to emulate.
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
+#include "eeconfig.h"
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum comet46_layers
+{
+    _QWERTY,
+    _RAISE,
+    _LOWER,
+    _ADJUST,
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+#define KC_RSE RAISE
+#define KC_LWR LOWER
+#define KC_ADJ ADJUST
+#define KC_CTTB CTL_T(KC_TAB)
+#define KC_CAD LCA(KC_DEL)
+#define KC_RST RESET
+
+#define LONGPRESS_DELAY 150
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+#define KC_ KC_TRNS 
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_QWERTY] = LAYOUT_kc(  
```

Comment:
Could you switch the default layout format, rather than the LAYOUT_kc?  

The LAYOUT_kc macro seems to cause problems for newer users.  It's fine for personal keymaps, or even a `default_kc` keymap. But for the normal default, we would like to avoid the _kc layouts.